test_container/tests/test/standard-flavor/all/import_modules.py

```diff
@@ -184,8 +185,8 @@ def run(ctx):
                 importer.import_modules()
             /
             '''))
-        with UdfDebugger(test_case=self):
-            rows = self.query('''SELECT scikit_learn.import_scikit_learn() FROM dual''')
+        #with UdfDebugger(test_case=self):
```

Comment:
```suggestion
```

Reply:
I left, that intentional, because it is likely, that this often need for debugging.